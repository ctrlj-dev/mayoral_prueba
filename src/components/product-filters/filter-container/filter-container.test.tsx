import { render } from '@testing-library/react';
import FilterContainer from './filter-container.component';

describe('FilterContainer specs', () => {
    test('test to check if container display his children properly', () => {
        //Arrange
        //Act
        const screen = render(<FilterContainer><p>filter1</p><p>filter2</p></FilterContainer>)

        const children1 = screen.getByText('filter1');
        const children2 = screen.getByText('filter2');

        expect(children1).not.toBeNull()
        expect(children2).not.toBeNull()

        expect(screen.asFragment()).toMatchSnapshot();

        //Assert
    })
})


