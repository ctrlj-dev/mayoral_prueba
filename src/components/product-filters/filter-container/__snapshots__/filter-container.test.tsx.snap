// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterContainer specs test to check if container display his children properly 1`] = `
<DocumentFragment>
  <div
    class="filterRow undefined row"
  >
    <p>
      filter1
    </p>
    <p>
      filter2
    </p>
  </div>
</DocumentFragment>
`;
